modules/liferay-courses-web/src/main/java/com/liferay/courses/web/portlet/action/EditCourseMVCActionCommand.java

```diff
@@ -0,0 +1,41 @@
+package com.liferay.courses.web.portlet.action;
+
+import com.liferay.courses.web.constants.LiferayCoursesPortletKeys;
+import com.liferay.portal.kernel.portlet.bridges.mvc.BaseMVCActionCommand;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCActionCommand;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferaybook.courses.api.LiferayCoursesAPI;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import javax.portlet.ActionRequest;
+import javax.portlet.ActionResponse;
+
+@Component (
+        immediate = true,
+        property = {
+                "javax.portlet.name=" + LiferayCoursesPortletKeys.PORTLET_ID,
+                "mvc.command.name=/courses/edit_course"
+        },
+        service = MVCActionCommand.class
+)
+
+public class EditCourseMVCActionCommand extends BaseMVCActionCommand {
+
+    /*I parametri del metodo ParamUtil.get... devono rispecchiare il campo input name presente nel file JSP.
+    La modifica viene accettata ugualmente ma i parametri modificati sono vuoti perchè il metodo non trova il campo che cerca.
+    * */
+    @Override
+    protected void doProcessAction(ActionRequest actionRequest, ActionResponse actionResponse){
+        Long courseId = ParamUtil.getLong(actionRequest, "courseId");
+        String name = ParamUtil.getString(actionRequest, "nome");
```

Comment:
Probabilmente sono i valori "nome" e "descrizione" che poi vengono visualizzati con iniziale minuscola nella view di Edit

Reply:
Nel caso in cui non venga specificato un campo label nel jsp, da automaticamente il nome, in lowercase, del componente stesso.